src/script/components/asset/controls/audioSeekBar.ts

```diff
@@ -16,41 +16,46 @@
  * along with this program. If not, see http://www.gnu.org/licenses/.
  *
  */
-
+import ko from 'knockout';
 import {debounce} from 'underscore';
 
 import {interpolate} from 'Util/ArrayUtil';
 import {clamp} from 'Util/NumberUtil';
 
+interface Params {
+  src: HTMLAudioElement;
+  disabled: ko.Subscribable<boolean>;
+
+  // TODO: replace with proper Type once they are defined
+  asset: {meta: {loudness: number[]}};
```

Comment:
If this is not properly typed, let's use `any` for the time being.
```suggestion
  asset: any;
```